servers/rendering/renderer_rd/shaders/effects/copy.glsl

```diff
@@ -241,12 +241,18 @@ void main() {
 
 #endif
 
-#ifdef MODE_LINEARIZE_DEPTH_COPY
+#if defined(MODE_LINEARIZE_DEPTH_COPY) || defined(MODE_LINEARIZE_DEPTH_COPY_INVERSED)
 
 	float depth = texelFetch(source_color, pos + params.section.xy, 0).r;
-	depth = depth * 2.0 - 1.0;
-	depth = 2.0 * params.camera_z_near * params.camera_z_far / (params.camera_z_far + params.camera_z_near - depth * (params.camera_z_far - params.camera_z_near));
+
+	depth = params.camera_z_near * params.camera_z_far /
+			(params.camera_z_far + depth * (params.camera_z_near - params.camera_z_far));
+
+#ifdef MODE_LINEARIZE_DEPTH_COPY_INVERSED
+	vec4 color = vec4(1.0 - (depth / params.camera_z_far));
+#else
```

Comment:
I wouldn't add a new shader version for this, I would just add a single boolean variable to the push constant and do:
```
vec4 color = vec4(depth / params.camera_z_far);
if (params.inverse_depth) {
    color = 1.0 - color;
}
```

Reply:
Ah, yeah that is definitely a better way of doing it, I'll change it to use a push constant value instead.